javascripts/discourse/components/tag-game-card-price.gjs

```diff
@@ -0,0 +1,59 @@
+import Component from "@glimmer/component";
+import { htmlSafe } from "@ember/template";
+import { bind } from "discourse-common/utils/decorators";
+
+export default class TagGameCardPrice extends Component {
+  constructor() {
+    super(...arguments);
+  }
+
+  @bind
+  getFormattedPrice(price) {
+    const priceString = String(price);
+    const [integerPart, decimalPart] = priceString.split(".");
+    const formattedIntegerPart = new Intl.NumberFormat("pt-BR").format(
+      integerPart
+    );
+    return htmlSafe(
+      `<span class="game-tag-card__price--highlighted"><sup style="top:-0.2rem"><small>R$</small></sup><span>${formattedIntegerPart}</span><sup style="top:-0.2rem"><small>,${
```

Comment:
This could be a separate component, to avoid this approach. In React this would be the same as using `dangerouslySetInnerHTML`